client/src/components/Post/index.tsx

```diff
@@ -83,50 +86,50 @@ export default class Post extends React.Component<IProps> {
         this.props.getPostsAsync(this.props.user.username);
     }
 
-    public onDeleteLike = (): void => {
+    public componentDidUpdate(prevProps: IProps): void {
+        if (this.props.userPosts.selectedPost.authorsOfLike !== prevProps.userPosts.selectedPost.authorsOfLike) {
+            this.props.setCountOfLikes(this.props.userPosts.selectedPost.authorsOfLike.length);
+
+            const arr = this.props.userPosts.selectedPost.authorsOfLike.filter((userId: string) =>
+                this.props.user._id === userId,
+            );
+
+            if (arr.length) {
+                return this.props.checkUserLikeExist(true);
+            }
+            return this.props.checkUserLikeExist(false);
```

Comment:
string 97 - 100 ------>

return this.props.checkUserLikeExist(!!arr.length);

Reply:
fixed